docs/Contribute-to-AlmaLinux-Build-System.md

```diff
@@ -0,0 +1,187 @@
+
+The commit message should meet the requirements:
+* Should contain short (72 chars or less) summary.
+* Should contain more detailed explanatory text. Wrap it to 72 characters. The blank line separating the summary from the body is critical (unless you omit the body entirely).
+* Further paragraphs come after blank lines.
+* Bullet points are okay, too.
+    * Typically a hyphen or asterisk is used for the bullet, followed by a single space. Use a hanging indent.
+* If want to add references to fixed or resolved issue(s), do it at the end as in the exmaple: 
+  ```
+  Fixes: https://github.com/AlmaLinux/build-system/issues/80.
+  ```
+  or 
+  ```
+  Resolves: https://github.com/AlmaLinux/build-system/issues/80.
+  ```
+
+Ideally, every commit message should look like this, but we try to be flexible enough.
+
+#### Summary
+
+Perfect summary should meet further requirements: 
+* Short commit title, 72 chars or less.
+* Commit summary, 72 chars or less per line.
+    * Keep it meaningful in a short way. Remember that you can add more details later.
+    * Using the imperative or simple past tenses is acceptable, but please be consistent.
+* Link a fixed/resolved issue:
+  ```
+  Fixes: https://github.com/AlmaLinux/build-system/issues/80
+  ```
+  or
+  ```
+  Resolves: AlmaLinux/build-system/issues/80
+  ```
+  It's optional, and can be used only when the commit message is short enough to fit in 72 characters.
+
+#### Body
+
+Body is a more detailed explanatory text. Wrap it to 72 characters. The blank line separating the summary from the body is critical (unless you omit the body entirely).
+
+Further paragraphs come after blank lines.
+
+* Bullet points are okay, too.
+* Typically a hyphen or asterisk is used for the bullet, followed by a single space. Use a hanging indent.
+
+When writing the body part, you can add as much details as you want on the commit message. Just remember:
+* The body is optional if the subject is enough to describe the commit.
+* Wrap lines to 72 characters.
+* Using the imperative or simple past tenses is acceptable, but please be consistent.
+* Feel free to describe what you think it's worth being described as if you were your colleague (that is more or less familiar to the code) that would like to understand what happened by just reading a few lines.
+
+#### Footer
+
+Footer is a perfect place to add references as links to fixed or resolved issue(s). If have not referenced issues in the subject, then they can be added here:
+```
+Fixes https://github.com/AlmaLinux/build-system/issues/80
+```
+or 
+```
+Resolves https://github.com/AlmaLinux/build-system/issues/80
+```
+
+Remember:
+* Not having referenced issues is a possibility in case of dependency version updates and so on.
+* Can use a bullet list for the list of references.
+* Although using Github's abbreviations is okay, we privilege using the full URL as it will be meaningful if somebody is not reading the commit message through Github's interface.
```

Comment:
```suggestion
* Although using Github's abbreviations is okay, we require using the full URL as it will be meaningful if somebody is not reading the commit message through Github's interface.
```